docs/api/rest_api/rest_api_reference/input/examples/product/catalog/attributes/GET/AttributeList.json.example

```diff
@@ -8,29 +8,32 @@
                 "name": "Attrib1",
                 "description": "",
                 "type": "Checkbox",
+                "type_identifier": "checkbox",
                 "group": "Attribute Group Test",
                 "position": 0,
-                "options": ""
+                "options": []
```

Comment:
A bit sideway question: Why this isn't `"options": {} ` as it seems to be an object when not empty? Empty array VS associative array encoding to JSON maybe?

Reply:
Yes, most likely. This comes from `Ibexa\Contracts\Rest\Output\Generator::valueElement` on `array<string,mixed>`.

Reply:
@konradoboza 

With all those cascades and recursion, it's not as easy as to add a `JSON_FORCE_OBJECT`… 

Anyway, it was a side remark.